### ISSUE TITLE
Type error in generated client

### ISSUE THREAD

**Issue by SilentRhetoric** [algorandfoundation/algokit-client-generator-ts#43]
I used `algokit generate client application.json -o client.generated.ts` and the generated client has a type error in the schema for global reserved state:

**Actual**
"descr"

```      "reserved": {
        "reserved_global_bytes_value": {
          "type": "bytes",
          "max_keys": 30,
          "descr": "Reserved global state bytes value with 30 possible keys"
        },
        "reserved_global_uint_value": {
          "type": "uint64",
          "max_keys": 29,
          "descr": "Reserved global state uint value with 29 possible keys"
        }
      }
```

**Expected**
"desc"

These are the types from algokit-utils:

```
/** Reserved schema value specification */
export interface ReservedSchemaValueSpec {
    /** The type of value */
    type: AVMType;
    /** The description of the reserved storage space */
    desc: string;
    /** The maximum number of slots to reserve */
    max_keys: number;
}
```

**Comment by achidlow** [comment]
@SilentRhetoric thanks for the report, I've transferred the issue to the repository where the problem would be.

cc @tristanmenzel

**Comment by tristanmenzel** [comment]
Looks like the typings in algokit-utils-ts are wrong. I'll update them there

**Comment by SilentRhetoric** [comment]
Reopening and I didn't realize this was transferred to the right repo.  Thank you guys!

**Comment by tristanmenzel** [comment]
This should be resolved in `@algorandfoundation/algokit-utils@2.3.0` and `@algorandfoundation/algokit-client-generator@2.2.3`.
